app/bundles/PageBundle/Views/FormTheme/Point/pointaction_urlhit_widget.html.twig

```diff
@@ -0,0 +1,107 @@
+<?php
+
+/*
+ * @copyright   2014 Mautic Contributors. All rights reserved
+ * @author      Mautic
+ *
+ * @link        http://mautic.org
+ *
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+$timeFrames = [
+    'i' => $view['translator']->trans('mautic.core.time.minutes'),
+    'H' => $view['translator']->trans('mautic.core.time.hours'),
+    'd' => $view['translator']->trans('mautic.core.time.days'),
+];
+
+?>
+
+<div class="row">
+    <div class="col-xs-12">
+        <?php echo $view['form']->row($form['page_url']); ?>
```

Comment:
@JoshuaEstes What's the plan with this template?

Reply:
No idea how I missed this

https://github.com/mautic/mautic/pull/12031